### PR TITLE
PDCT-283 - search on geo and doc pages

### PR DESCRIPTION
**Pull request by PatFawbertMills**
- Header search bar expands with feature searches and a 'try this' section
- A help indicator / tooltip-style will appear for new users to point them towards the search bar
- Update to the header and footer UI
- Update to the geography page UI
- Update to the family document page UI

**Important note:** The version of node in the dockerfile has been updated as the previous version of `16-alpine3.13` was incompatible with the latest npm, it is now `20-alpine3.17`

### CODE REVIEW

File: src/components/FloatingSearch.tsx [hunk 1]
```diff
@@ -1,38 +1,152 @@
...
+    const queryObj = CleanRouterQuery({ ...router.query });
+    queryObj[QUERY_PARAMS.query_string] = term;
+    router.push({ pathname: "/search", query: queryObj });
   };
 
+  const handleTryThisClick = (e: any) => {
+    e.preventDefault();
+    const queryObj = {
+      [QUERY_PARAMS.query_string]: "Flood defence",
+      [QUERY_PARAMS.country]: "czechia",
```
Comment (diversemix) [inline src/components/FloatingSearch.tsx:41]: Is this hardcoded data meant to be here?

File: src/components/FloatingSearch.tsx [hunk 2]
```diff
@@ -1,38 +1,152 @@
...
+                    }}
+                    className="text-gray-800 bg-white border border-gray-300 rounded-[40px] py-1 px-2 transition hover:bg-blue-600 hover:text-white"
+                  >
+                    {searchTerm}
+                  </button>
+                </li>
+              ))}
+            </ul>
+          </div>
+          <a
```
Comment (diversemix) [inline src/components/FloatingSearch.tsx:112]: Maybe it was meant to be there :laughing:

Comment (PatFawbertMills) [reply src/components/FloatingSearch.tsx:112]: Yes it is intended.

This is a good example of what I was talking about previously regarding some basic CMS features to populate certain pieces of content.

**Review by diversemix — APPROVED** [review]
lgtm

**Pull request merged**